public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -105,97 +106,104 @@ export class PanelQueryRunner {
     }
 
     return this.subject.pipe(
-      this.getTransformationsStream(withTransforms),
-      map((data: PanelData) => {
-        let processedData = data;
-        let streamingPacketWithSameSchema = false;
+      mergeMap((data: PanelData) => {
+        if (data.series === lastRawFrames) {
+          console.log('returning same series');
+          return of({ ...data, series: lastProcessedFrames });
```

Comment:
so returning new data panel instance (before loading state), but same frames array

Reply:
🤔  this may actually bypass the complex isStreamingDataFrame logic below

Reply:
should be:

```suggestion
          return of({ ...data, structureRev, series: lastProcessedFrames });
```